src/Exceptions/DecryptException.php

```diff
@@ -0,0 +1,20 @@
+<?php declare(strict_types=1);
+namespace Healthlabs\Sodium\Exceptions;
+
+use Throwable;
+
+/**
+ * The exception to be thrown when decryption failed.
+ */
+class DecryptException extends SodiumException
+{
+    const message = 'Unable to decrypt the payload using the given key';
+
+    public function __construct($message = '', $code = 0, Throwable $previous = null)
+    {
+        if (empty($message)) {
+            $message = self::message;
+        }
+        parent::__construct($message, $code, $previous);
+    }
```

Comment:
You can remove the if with something like ... 



```suggestion
    public function __construct(?string $message = '', int $code = 0, Throwable $previous = null)
    {
        parent::__construct($message ?? self::message, $code, $previous);
    }
```

Also, i dunno if is in our standards, but all constants should be uppercase right?

Reply:
Can't use null coalescing operator here since the $message won't be null, I used ternary instead.